src/components/Header/Header.test.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import Header, { Logo } from './index';
+import { configuration } from '../../config';
+import Img from '../Img';
+
+const HeaderWrapper = props => (
+  <MemoryRouter>
+    <Provider store={props.store}>
+      <Header
+        {...props}
+      />
+    </Provider>
+  </MemoryRouter>
+);
+
+jest.mock('@edx/frontend-platform/auth', () => ({
+  getAuthenticatedUser: jest.fn(),
+  hydrateAuthenticatedUser: jest.fn(),
+}));
+
+HeaderWrapper.propTypes = {
+  store: PropTypes.shape({}).isRequired,
+};
+
+describe('<Logo />', () => {
+  it('renders enterprise logo correctly', () => {
+    const props = {
+      enterpriseLogo: 'https://test.url/image/1.png',
+      enterpriseName: 'Test Enterprise',
+    };
+
+    const wrapper = mount(<Logo {...props} />);
```

Comment:
nit: would love to get rid of enzyme at some point but step by step! not for this PR already quite big!

Reply:
We'll get there :)

Reply:
yeah I love this discussion! Sure we should def bring up ideas and chat about this area! Maybe build up/improve testing docs as we work on new areas and doc/recommend where enzyme may be better suited. my default take is always to go RTL since it forces you to think a bit differently focusing only on what users can see or interact with. It also seems to try to prevent you mostly from trying to access internal state etc. Which is great in my opinion. And we have hooks testing lib for most of the rest, plus plain jest tests for utils. That may leave some balance/room though where RTL does not do well, and we can identify and use enzyme for those. I still make it a point to largely stay away from enzyme these days unless there is a compelling reason to use it (I am certain those reasons exist in places!)

Reply:
Sounds like a good potential discussion topic for a future Enterprise Architecture Working Group office hours? :)

Reply:
Yeah, and I'm super on board with the way `react-testing-library` makes you think about things. It's great in that way. 
And yeah, in our repos, I think we should lean heavily towards it. I think enzyme shines more when you're creating a generic reusable component that you need to test is passing props around as it should. Serious unit testing. :)